guides/source/active_record_basics.md

```diff
@@ -142,7 +144,7 @@ end
 This will create a `Product` model, mapped to a `products` table at the
 database. By doing this you'll also have the ability to map the columns of each
 row in that table with the attributes of the instances of your model. Suppose
-that the `products` table was created using an SQL statement like:
+that the `products` table was created using an MySQL statement like:
```

Comment:
Can you change the article before `MySQL` ("an" to "a")? It's incorrect either way so we might as well improve the docs while in here.

Reply:
Done.